### PR TITLE
[Feature Store] Fix mix-up of transient time columns between multiple spark targets

### PR DESCRIPTION
**Pull request by gtopper**
ML-1908

### CODE REVIEW

File: tests/system/feature_store/test_spark_engine.py [hunk 1]
```diff
@@ -27,9 +27,11 @@ class TestFeatureStoreSparkEngine(TestMLRunSystem):
     pq_source = "testdata.parquet"
     csv_source = "testdata.csv"
     spark_image_deployed = (
-        False  # Set to True if you want to avoid the image building phase
+        True  # Set to True if you want to avoid the image building phase
```
Comment (benbd86) [inline tests/system/feature_store/test_spark_engine.py:30]: should that default be changed?

Comment (benbd86) [inline tests/system/feature_store/test_spark_engine.py:30]: should that default be changed?

File: tests/system/feature_store/test_spark_engine.py [hunk 2]
```diff
@@ -27,9 +27,11 @@ class TestFeatureStoreSparkEngine(TestMLRunSystem):
...
     )
-    test_branch = ""  # For testing specific branch. e.g.: "https://github.com/mlrun/mlrun.git@development"
+    test_branch = "https://github.com/gtopper/mlrun.git@ML-1908"
```
Comment (benbd86) [inline tests/system/feature_store/test_spark_engine.py:32]: return to default

Comment (benbd86) [inline tests/system/feature_store/test_spark_engine.py:32]: return to default

**Review by benbd86 — APPROVED** [review]
just restore the defaults

**Review by Hedingber — APPROVED** [review]

**Pull request merged**